tck/src/main/java/org/reactivestreams/tck/TestEnvironment.java

```diff
@@ -1025,22 +1072,44 @@ public void expectCompletion(long timeoutMillis, String errorMsg) throws Interru
       } // else, ok
     }
 
-    @SuppressWarnings("unchecked")
+    /**
+     * @deprecated Deprecated in favor of {@link #expectError(Class, long, long, String)}.
+     */
+    @Deprecated
     public <E extends Throwable> E expectError(Class<E> clazz, long timeoutMillis, String errorMsg) throws Exception {
```

Comment:
deprecating sounds good; noone should have really touched those methods outside of the TCK itself I suppose... though yeah, let's play nice and not remove the method :)

Reply:
yip ... bunch of methods look over exposed here, but I don't want to break anything.